src/XRP/wallet-factory.ts

```diff
@@ -38,6 +38,7 @@ export default class WalletFactory {
    * @param derivationPath - The given derivation path to use. If undefined, the default path is used.
    * @returns A new wallet from the given mnemonic if the mnemonic was valid, otherwise undefined.
    */
+  // eslint-disable-next-line max-statements -- This does not read as too complex a function.
```

Comment:
We have a limit of 10 statements per function.

Possibly the following could actually be pulled out into its own function like `getBip32Node()`, and then we wouldn't need this disable?

```ts
const seedBytes = Utils.toBytes(seed)
const seedBuffer = Buffer.from(seedBytes)
const masterNode = bip32.fromSeed(seedBuffer)
const node = masterNode.derivePath(derivationPath)
if (node.privateKey === undefined) {
  return undefined
}
```